lib/reverse_sentence.rb

```diff
@@ -1,6 +1,22 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)? because we are doing two loops but we drop the constants
+# Space complexity: O(1) -- I don't feel confident about this because of the creation of split_sentence and/or reversed
 def reverse_sentence(my_sentence)
```

Comment:
This works, but your space complexity is O(n) because you split the string into an array (creating an array of all the words), further there is an alternative solution which is also O(n) time complexity and O(1) space complexity.

Consider: Reverse the string completely. Then loop through the string reversing each individual word, in place (i.e. not making an array). Think about how you might be able to do that.